src/tests/readytorun/JittedMethodsCountingTest/JittedMethodsCountingTest.cs

```diff
@@ -0,0 +1,129 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using Xunit;
+
+public class JittedMethodsCountingTest
+{
+    private const int MAX_JITTED_METHODS_ACCEPTED = 120;
+
+    [Fact]
+    public static int TestEntryPoint()
+    {
+        // If DOTNET_ReadyToRun is disabled, then this test ought to be skipped.
+        if (!IsReadyToRunEnvSet())
+        {
+            Console.WriteLine("\nThis test is only supported in ReadyToRun scenarios."
+                              + " Skipping...\n");
+            return 100;
+        }
+
+        string appName = "HelloWorld.dll";
+        string jitOutputFile = "jits.txt";
```

Comment:
I think you should generate a full path akin to what I did in #90647,

```
      string testAppFolder = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);

        string appName = Path.Combine(testAppFolder, "HelloWorld.dll");
        string jitOutputFile = Path.Combine(testAppFolder, "jits.txt");
```

The scripts may be tampering with the current directory.

Reply:
Thanks for the suggestion Tomas! Will try it this way.